include/h2o/absprio.h

```diff
@@ -54,8 +47,9 @@ static uint16_t h2o_absprio_urgency_to_chromium_weight(uint8_t urgency);
 inline uint16_t h2o_absprio_urgency_to_chromium_weight(uint8_t urgency)
 {
     uint16_t weight;
-    assert(urgency < 8);
-    weight = (8 - urgency) * 32;
+    assert(urgency < H2O_ABSPRIO_NUM_URGENCY_LEVELS);
+    weight = (H2O_ABSPRIO_NUM_URGENCY_LEVELS - urgency) * 32;
```

Comment:
@hfujita Do we need to change this mapping now that the default level of urgency has changed from 1 to 3?

Reply:
That's a good question. I think we might need to reconsider this mapping logic as a whole, as IIRC Chromium does not use this kind of linear weight distribution i.e. I think I saw values like 256, 220, 147, etc., which are not necessarily 32*N.

Do you happen to know a good source of what Chromium (Chrome) would use as a weight value? I see [this](https://chromium.googlesource.com/chromium/blink/+/master/Source/core/fetch/ResourceFetcher.cpp#63) but `ResourceLoadPriority` is just a [sequential enum from 0 to 6](https://chromium.googlesource.com/chromium/blink/+/master/Source/platform/network/ResourceLoadPriority.h), so there must be another logic to convert this value into H2 weight (which I haven't found yet).

Reply:
I think it might be https://quiche.googlesource.com/quiche/+/refs/heads/master/spdy/core/spdy_protocol.cc#50 now that they've split the protocol implementation as a separate repository? cc @janaiyengar

Reply:
This looks promising, thanks for the pointer. I'll take a closer look and will submit another PR.

Reply:
PR #2521 created.